modules/op-understanding-credential-selection.adoc

```diff
@@ -0,0 +1,42 @@
+// This module is included in the following assembly:
+//
+// *openshift-docs/cicd/pipelines/authenticating-pipelines-using-git-secret.adoc
+
+[id="op-understanding-credential-selection_{context}"]
+= Understanding credential selection
```

Comment:
IBM SG: "If the subject is a functional overview, begin a heading with words such as Introduction or Overview rather than contriving a pseudo-task-oriented heading that begins with Understanding, Using, Introducing, or Learning."
```suggestion
= Credential selection
```